rx2dnssd/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
 
     testImplementation 'junit:junit:4.13.2'
     // Last version that support Power Mock: https://github.com/powermock/powermock/wiki/Mockito#supported-versions
-    testImplementation 'org.mockito:mockito-core:4.8.0'
+    testImplementation 'org.mockito:mockito-core:3.6.0'
```

Comment:
What's the reason for downgrading Mockito?

Reply:
Because as specified in the comment above, version 3.6.0 seems to be the last one that correctly supports PowerMock. If you upgrade to version 4.8.0, the unit tests stop working.